EHR_Purchasing/resources/queries/ehr_purchasing/purchasingRequests.query.xml

```diff
@@ -0,0 +1,34 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="purchasingRequests" tableDbType="TABLE" useColumnOrder="true">
+                <tableTitle>Purchasing Requests</tableTitle>
+                <columns>
+                    <column columnName="vendorId">
+                        <columnTitle>Vendor</columnTitle>
+                        <fk>
+                            <fkDbSchema>ehr_purchasing</fkDbSchema>
+                            <fkTable>vendor</fkTable>
+                            <fkColumnName>rowId</fkColumnName>
```

Comment:
You can omit the fkColumnName and it will default to using the PK of the target table. Similarly, it's typically best to set the titleColumn on the target table instead of specifying it as the fkDisplayColumnName every time it's referenced.

Reply:
good points, done!